packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -123,6 +129,22 @@ export class Copy {
         }
     }
 
+    /**
+     * Function that checks if the data set exists
+    **/
+    public static async dataSetExists(
```

Comment:
If this method isn't used outside of this file, we may want to make it private for now.